src/services/admin/mentor.service.test.ts

```diff
@@ -0,0 +1,78 @@
+import { startServer } from '../../app'
+import type { Express } from 'express'
+import supertest from 'supertest'
+import { dataSource } from '../../configs/dbConfig'
+import { mockAdmin, mockMentor, mentorApplicationInfo } from '../../../mocks'
+import bcrypt from 'bcrypt'
+import Profile from '../../entities/profile.entity'
+import { ApplicationStatus, ProfileTypes } from '../../enums'
+import { getAllMentors } from './mentor.service'
+import type { AllMentorsArray } from '../../types'
+
+const port = Math.floor(Math.random() * (9999 - 3000 + 1)) + 3000
+
+let server: Express
+let mentorAgent: supertest.SuperAgentTest
+let adminAgent: supertest.SuperAgentTest
+let mentorId: string
+
+describe('getAllMentors function', () => {
```

Comment:
This should be route testing. Move to the `src/routes/admin/mentor/mentor.route.test.ts` file